frontend-public/src/styles/settings/themeOverride.scss

```diff
@@ -28,3 +24,34 @@
     border: 1px solid $primary-btn-border-color-hover;
   }
 }
+
+.ant-form-item-label {
+  label {
+    font-size: $default-p;
+    color: $p-color;
+    font-weight: $semi-bold;
+  }
+}
+
+.ant-modal-title {
+  font-size: $default-h2;
+  color: $p-color;
+  font-weight: $semi-bold;
+}
+
+.ant-radio-button-wrapper {
```

Comment:
This is an ant class correct?? 
if so, the `!importants` used in this situation are not alarming compared to the other situation.
but is the CSS not being overwritten without them?

Reply:
Without the important it leaves the right side of the border the wrong colour